functions/Get-DbaSqlManagementObject.ps1

```diff
@@ -62,22 +62,24 @@
 		$scriptblock = {
 			$VersionNumber = [int]$args[0]
 			
-			Write-Verbose "Checking currently loaded SMO version"
+			Write-Message -Level Verbose -Message "Checking currently loaded SMO version"
 			$loadedversion = [AppDomain]::CurrentDomain.GetAssemblies() | Where-Object { $_.Fullname -like "Microsoft.SqlServer.SMO,*" }
 			
 			if ($loadedversion)
 			{
 				$loadedversion = $loadedversion | foreach { ((Split-Path (Split-Path $_.Location) -Leaf) -split "__")[0] }
 			}
 			
-			Write-Verbose "Looking for SMO in the Global Assembly Cache"
+			Write-Message -Level Verbose -Message "Looking for included smo library"
+			$localversion = [version](Get-ChildItem -Path "$script:PSModuleRoot\bin\Microsoft.SqlServer.Smo.dll").VersionInfo.ProductVersion
```

Comment:
hey @dbaduck - you can use this to parse the local version 👍 If you'd just like to add on to this PR, let me know.